app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cd, old_file, new_file = command.split()
+    except ValueError:
+        raise ValueError("Write a command correctly. "
+                         "Example: cp file1.txt file2.txt")
+    if old_file == new_file:
+        return
+
+    with open(old_file, "r") as file_in, open(new_file, "w") as file_out:
+        file_out.write(file_in.read())
```

Comment:
You can combine condition and with block here. Definitely mentor Yurii would reject this.I don't know why but we shouldn't make blank return without huge reason. Got several rejects cause of it.